kalite/distributed/static/css/distributed/sidebar.css

```diff
@@ -134,7 +131,20 @@ div.fade {
     color: #CCC;
 }
 
-.back-to-parent {
+.sidebar-back {
+    height: 100%;
+    width: 45px;
+    padding-top: 100px;
+    text-align: center;
+    color: #fff;
+    background-color: #467471;
+    font-size: 24px;
+    border-right: 1px solid rgba(0, 0, 0, 0.3);
+    position: absolute;
+    z-index: 10000;
+}
+
+/*.back-to-parent {
```

Comment:
`.back-to-parent`? They have been in there for a while. There was never a back button implemented. They can be removed now that I have a working back button. I'll clean that stuff up. It's used in some JS files too.